spinoso-time/src/lib.rs

```diff
@@ -63,69 +70,3 @@ pub const NANOS_IN_SECOND: u32 = Duration::from_secs(1).as_nanos() as u32;
 /// Number of microseconds in one nanosecond.
 #[allow(clippy::cast_possible_truncation)] // 1000 < u32::MAX
 pub const MICROS_IN_NANO: u32 = Duration::from_micros(1).as_nanos() as u32;
-
-/// Error returned when constructing a [`Time`] from a [`ToA`].
-///
-/// This error is returned when a time component in the `ToA` exceeds the maximum
-/// permissible value for a datetime. For example, invalid values include a
-/// datetime 5000 days or 301 seconds.
-///
-/// # Examples
-///
-/// Invalid date component:
-///
-/// ```
-/// # use spinoso_time::{Offset, Time, ToA, ComponentOutOfRangeError};
-/// let to_a = ToA {
-///     sec: 21,
-///     min: 3,
-///     hour: 23,
-///     day: 5000,
-///     month: 4,
-///     year: 2020,
-///     wday: 0,
-///     yday: 96,
-///     isdst: true,
-///     zone: Offset::Local,
-/// };
-/// let time = Time::try_from(to_a);
-/// assert_eq!(time, Err(ComponentOutOfRangeError::Date));
-/// ```
-///
-/// Invalid time component:
-///
-/// ```
-/// # use spinoso_time::{Offset, Time, ToA, ComponentOutOfRangeError};
-/// let to_a = ToA {
-///     sec: 301,
-///     min: 3,
-///     hour: 23,
-///     day: 5,
-///     month: 4,
-///     year: 2020,
-///     wday: 0,
-///     yday: 96,
-///     isdst: true,
-///     zone: Offset::Local,
-/// };
-/// let time = Time::try_from(to_a);
-/// assert_eq!(time, Err(ComponentOutOfRangeError::Time));
-/// ```
-#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, PartialOrd, Ord)]
-pub enum ComponentOutOfRangeError {
```

Comment:
Resolving because of later comment :smile:. I think it might still be possible to make and/or unify a bunch of errors from this crate. It feels like it'll be easier later - I'll create a follow up issue for it.